### PR TITLE
Update test data file

### PR DESCRIPTION
**Pull request by victoria34**
Update test example files for new data release

### CODE REVIEW

File: core/src/test/java/org/mskcc/cbio/oncokb/util/FindRelevantAlterationsTest.java [hunk 1]
```diff
@@ -53,7 +53,7 @@ public FindRelevantAlterationsTest(String hugoSymbol, String alteration, String
                 // General truncating consequence should be associated with Truncating Mutations
                 // Check splice
                 // TP53 Oncogenic Mutations does not have any information we are ready to relase
-                {"TP53", "X33_splice", null, "X33_splice, Truncating Mutations"},
+                {"TP53", "X33_splice", null, "X33_splice, Truncating Mutations, Oncogenic Mutations"},
```
Comment (zhx828) [inline core/src/test/java/org/mskcc/cbio/oncokb/util/FindRelevantAlterationsTest.java]: This seems that you did not run the sql script. These Oncogenic Mutations should not be there

File: core/src/test/resources/test_treatments.txt [hunk 2]
```diff
@@ -74,8 +74,8 @@ KIT	D816A	Thymic Tumor	2: Sunitinib; 3B: Regorafenib; 3B: Sorafenib; 3B: Avaprit
 MTOR	L2427Q	Renal Clear Cell Carcinoma	3A: Temsirolimus; 4: Everolimus;
 PDGFRA	D842I	GIST	2: Imatinib;
 PDGFRA	D842I	Thymic Tumor	3B: Imatinib;
-PDGFRA	D842V	GIST	R1: Imatinib; 2: Dasatinib;
-PDGFRA	D842V	Thymic Tumor	3B: Dasatinib; 3B: Imatinib;
+PDGFRA	D842V	GIST	R1: Imatinib; 1: Avapritinib; 2: Dasatinib;
```
Comment (zhx828) [inline core/src/test/resources/test_treatments.txt]: These look correct.

File: core/src/main/java/org/mskcc/cbio/oncokb/util/SummaryUtils.java [hunk 3]
```diff
@@ -127,7 +128,7 @@ public static String variantCustomizedSummary(Set<Gene> genes, Alteration exactM
         alternativeAlleles.add(alteration);
         alternativeAlleles.addAll(AlterationUtils.getPositionedAlterations(alteration));
 
-        alternativeAlleles = (List<Alteration>) CollectionUtils.intersection(alternativeAlleles, relevantAlterations);
+        alternativeAlleles = ListUtils.intersection(alternativeAlleles, relevantAlterations);
```
Comment (victoria34) [inline core/src/main/java/org/mskcc/cbio/oncokb/util/SummaryUtils.java]: Type conversion will change the list order.

**Comment by zhx828** [comment]
@victoria34 remember do a squash merge when it's ready

**Pull request merged**